src/ui/Button/Button.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { FaSearch } from "react-icons/fa";
 
-import './BtnSearch.scss';
+import './Button.scss';
 
-const BtnSearch = ({ type = "submmit" }) => (
+const Button = ({ type = "submmit" }) => (
   <button
     type={type}
-    className="search__btn">
+    className="button__search">
```

Comment:
I think the className can be just button :thinking: if we have a modifier for the search `button--search` as well, but just for the things that are specific for search